src/liquid/app/form_switch.liquid

```diff
@@ -0,0 +1,61 @@
+<div id="place_form_parent">
+  {% capture header_caption %}
+    {% if op == "bulk" %} Bulk Load from CSV
+    {% elsif op == "replace" %} Replace Existing
+    {% elsif op == "move" %} Move
+    {% elsif op == "merge" %} Merge Two
+    {% elsif op == "delete" %} Delete a
+    {% elsif op == "edit" %} Edit
+    {% else %} New
+    {% endif %}
+  {% endcapture %}
```

Comment:
Consider using `case .. when` , IMO it will read better, for this and the other sections that have lots of `if.. else`

```suggestion
{% capture header_caption %}
    {% case op %}
      {% when "bulk"   %} Bulk Load from CSV
      {% when "replace"%} Replace Existing
      {% when "move"   %} Move
      {% when "merge"  %} Merge Two
      {% when "delete" %} Delete a
      {% when "edit"   %} Edit
      {% else             %} New
    {% endcase %}
  {% endcapture %}
```